src/api/surahs.ts

```diff
@@ -0,0 +1,77 @@
+import axios, { AxiosResponse } from 'axios';
+import Lesson from '@/models/lesson/lesson';
+import Surah from '@/models/surah/surah';
+import buildUrl from './buildUrl';
+
+async function getSurahs(): Promise<Surah[]> {
+  try {
+    const response: AxiosResponse = await axios.get(buildUrl('/surahs'));
+    
+    const surahs: Surah[] = [];
+
+    response.data.data.forEach((responseSurah: any) => {
```

Comment:
@osamaramihafez we also should probably not put the response in a nested "data" like we are here.

Reply:
Yeah, once we've figured out the typing and packaging I'm down to take that out insha'Allah or rename it. Any suggestions?

Reply:
I just meant, needing to do `response.data.data` is useless. Should just be `response.data`